sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonEditForm.java

```diff
@@ -631,6 +648,100 @@ protected void addFields() {
 			I18nProperties.getPrefixDescription(PersonDto.I18N_PREFIX, PersonDto.ADDITIONAL_DETAILS, "") + "\n"
 				+ I18nProperties.getDescription(Descriptions.descGdpr));
 		CssStyles.style(additionalDetails, CssStyles.CAPTION_HIDDEN);
+
+		hasGuardian = addField(PersonDto.HAS_GUARDIAN, CheckBox.class);
+		hasGuardian.addValueChangeListener(e -> updateHasGuardianCheckBox());
+
+		isIncapacitated = addField(PersonDto.IS_INCAPACITATED, CheckBox.class);
+		isIncapacitated.addValueChangeListener(e -> updateIsIncapacitatedCheckBox(true));
+
+		isEmancipated = addField(PersonDto.IS_EMANCIPATED, CheckBox.class);
+		isEmancipated.addValueChangeListener(e -> updateIsEmancipatedCheckBox());
+		hasGuardian.setEnabled(false);
+		minimumAdultAge = FacadeProvider.getConfigFacade().getMinimumAdultAge();//2 places
+		minimumEmancipatedAge = FacadeProvider.getConfigFacade().getMinimumEmancipatedAge();
+	}
+
+	private boolean isMinimumAgesInterval() {
```

Comment:
this method identifies if the person has the age within the emancipated interval, please rename it with a more meaningful name. An example, could be renamed to "canBeEmancipated"